helm/hpcc/templates/network.yaml

```diff
@@ -0,0 +1,71 @@
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: default-deny-all
+spec:
+  podSelector: {}
+  policyTypes:
+  - Ingress
+  - Egress
+  egress:
+  - to:
+#  This only works if we know that the kube-system namespace has been labelled, which we cannot control?
+    - namespaceSelector:
+        matchLabels:
+          name: kube-system
+---
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1 
+metadata:
+  name: allow-apiserver
+spec:
+  policyTypes:
+  - Egress
+  podSelector: {}
+  egress:
```

Comment:
Need to allow egress to the server used by kubectl, for components that use it. However the cidr to specify will vary from system to system.

Alternatives:
1. Don't block egress by default
2. Allow helm user to specify kubectl server to unblock, if they want to block egress